scripts/presubmit.sh

```diff
@@ -87,7 +87,7 @@ main() {
     echo 'running go build'
     go build ./...
 
-    export TEST_FLAGS="-short -timeout=${GO_TEST_TIMEOUT:-5m}"
+    export TEST_FLAGS="-timeout=${GO_TEST_TIMEOUT:-5m}"
```

Comment:
It might be useful to be able to supplement these flags from outside of the script, e.g.
```suggestion
    export TEST_FLAGS="${TEST_FLAGS} -timeout=${GO_TEST_TIMEOUT:-5m}"
```

Reply:
That might be best done as a separate change though, and would call for a CHANGELOG entry since there's a chance someone already has that environment variable set and it suddenly starts interacting with this script.

Reply:
Will leave as-is. If anyone wants this feature they can add it. It's orthogonal to the thrust of this PR.

Reply:
The relevance was that it'd allow people to keep running the tests with the -short flag if they so desired.